qsirecon/workflows/recon/anatomical.py

```diff
@@ -107,14 +92,15 @@ def init_highres_recon_anatomical_wf(
         f"Found high-res anatomical data in preprocessed inputs for {subject_id}."
     )
     workflow.connect([
-        (anat_ingress_node, outputnode,
-            [(name, name) for name in qsiprep_highres_anatomical_ingressed_fields])])  # fmt:skip
+        (inputnode, outputnode, [
+            (name, name) for name in qsiprep_highres_anatomical_ingressed_fields
+        ]),
+    ])  # fmt:skip
 
-    # grab un-coregistered freesurfer images later use
-    if subject_freesurfer_path is not None:
-        status["has_freesurfer"] = True
+    # grab un-coregistered freesurfer images for later use
+    if status["has_freesurfer"]:
```

Comment:
We'll have to make this fancier someday so it can look for session-specific freesurfers